srcpkgs/font-weather-icons/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'font-weather-icons'
+pkgname=font-weather-icons
+version=2.0.10
+revision=1
+archs=noarch
+create_wrksrc=yes
```

Comment:
This shouldn't be necessary.

Reply:
@abenson how do i go about removing it? if i remove `create_wrksrc` I get
```
/void-packages/common/xbps-src/shutils/common.sh: line 31: cd: /builddir/font-weather-icons-2.0.10: No such file or directory
```

Reply:
Oh, I think i've got it - should I manually specify `wrksrc` to be `"weather-icons-${version}"` and change `do_install` accordingly?

Reply:
Yes, I think that's preferred.

Reply:
Ok, I've pushed some changes to that effect